--- ts41x.lua.orig	2013-01-06 20:35:09.000000000 +0100
+++ ts41x.lua	2013-01-06 21:52:55.928116829 +0100
@@ -35,5 +35,9 @@
 end
 
 function temp( temp )
-	print("ts41x temperature:", temp)
+	if math.fmod( os.time(), 60 ) < 6 then
+		local file = assert(io.open("/var/cache/cacti/qcontrol.tmp", "w"))
+		file:write( string.format("acpitz:0 core0:%d core1:0 core2:0 core3:0\n", temp) )
+		file:close()
+	end
 end
